configure.ac

```diff
@@ -88,7 +88,7 @@ AC_TYPE_UID_T
 # ----------------------------------------------------------------------
 AC_FUNC_CLOSEDIR_VOID
 AC_FUNC_STAT
-AC_CHECK_FUNCS([fstatat memmove readlinkat strdup strncasecmp strstr])
+AC_CHECK_FUNCS([fstatat memmove openat readlinkat strdup strncasecmp strstr])
```

Comment:
What about splitting this list one function per line?

Reply:
done